less/_components/mobile-sidebar.less

```diff
@@ -0,0 +1,61 @@
+
+#mobile-sidebar {
```

Comment:
Do we think we will need to re-use these styles more than once on a page? If so, we should make this a class instead of an id!

Reply:
I think this is only for mobile sidebar

Reply:
Change it to classname  .mobile-sidebar